source/manual/post-a-statuspage-message.html.md

```diff
@@ -66,7 +72,10 @@ Example messages:
 
 ### "Monitoring" status
 
-We've fixed the issue - but are keeping are an eye on it to make sure it's properly resolved. State whether there will be a delay to the fix being visible, ie. "This should start functioning normally again in 20 minutes."
+Example messages:
+
+We've fixed the issue - but are keeping an eye on it to make sure it's properly resolved.
```

Comment:
This needs to be in a `>` block.

Reply:
This is not an example message; it's the explanation. I had wrongly added `Example messages:` above it when it should have been below it. I moved it now.

Reply:
Oh cool!